.travis.yml

```diff
@@ -13,6 +13,8 @@ branches:
 before_install:
 - echo no | android create avd --force -n test -t $ANDROID_TARGET --abi $ANDROID_ABI
   -c 20M
+- echo yes | android update sdk --filter extra-android-support --no-ui --force > /dev/null
```

Comment:
Travis recommends doing these updates through the ```components``` section.

I'd give it a try to add ``` - extra-android-m2repository``` there, and maybe also ```- extra-android-support```